### PR TITLE
Alisakina99 mandatorymfaplanning

### PR DESCRIPTION
**Pull request by alisakina99**
## Description
Please provide a detailed description of your changes.

## Related Issue
Link to any related issues or discussions here. This helps reviewers understand the context and the need for your changes.

## Checklist

- [x] I've tested my changes to ensure they are ready for review.
- [x] I've read the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md) guide.
- [x] I've updated the documentation (if applicable).
- [x] Resource Graph queries have been included (and tested) for recommendations where ever possible[^note].
- [x] Resource Graph queries have NOT been included (please explain below).

## Additional Information
Is there any additional context, screenshots, or considerations that might help in the review process? Please include them here.

## Reviewer Notes
Is there a specific area you’d like feedback on? Please highlight it here. We're here to help and learn together! 💡

[^note]:
    Details on how to add Azure Resource Graph queries to recommendations can be found [here](../blob/main/CONTRIBUTING.md#1-adding-or-modifying-resource-graph-queries).

### CODE REVIEW

File: checklists/alz_checklist.en.json [hunk 1]
```diff
@@ -218,13 +218,13 @@
         {
             "category": "Identity and Access Management",
             "subcategory": "Identity",
-            "text": "Enforce multi-factor authentication for any user with rights to the Azure environments.",
+            "text": "Starting in 2024, multifactor authentication (MFA) will be mandatory for all Azure sign-in attempts. Plan for and enforce multi-factor authentication for any user with rights to the Azure environments.",
```
Comment (brsteph) [inline checklists/alz_checklist.en.json:221]: Can you explain why this change is needed?  It moves the recommendation text itself further into the field and makes the expected action.

Comment (igorjnzl) [reply checklists/alz_checklist.en.json:221]: @alisakina99 we can leave this one as-is, it relates more to general MFA hygiene. 

The update in B03.14 will be sufficient to raise awareness around the changes starting in Oct 2024 around mandatory MFA

**Review by igorjnzl — APPROVED** [review]
